lib/cell/templates.rb

```diff
@@ -30,7 +30,7 @@ def create(prefix, view, options)
     # {["comment/row/views", comment/views"]["show.haml"] => "Tpl:comment/view/show.haml"}
     class Cache
       def initialize
-        @store = {}
+        @store = Hash.new({})
```

Comment:
This is problematic as the default here is a single instance of `{}` and is reused for *all* calls. The block form exists to work around this issue.

```ruby
irb(main):001:0> store = Hash.new({})
=> {}
irb(main):002:0> store[:foo][:bar] = 1
=> 1
irb(main):003:0> store[:baz]
=> {:bar=>1}
```